helm/chart-operator/templates/deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         app: {{ .Values.project.name }}
         giantswarm.io/service-type: "managed"
       annotations:
-        releasetime: {{ $.Release.Time }}
+        releaseRevision: {{ $.Release.Revision | quote }}
```

Comment:
Release.Time got removed in Helm 3 but Release.Revision still exists.
https://github.com/helm/helm/issues/5732

We use this so pods are rolled when the release is updated. So revision also works.